tests/test_spacecurve_casting.py

```diff
@@ -0,0 +1,29 @@
+import jax.numpy as jnp
+import numpy as np
+from qurveros.spacecurve import SpaceCurve
+
+def test_set_params_casts_int_to_float():
```

Comment:
Please convert these test functions into unit tests, following the format of the other test files. In particular, see [test_qubit_bench.py](https://github.com/error-corp/qurveros/blob/1f53758e14e6be53cec63a159eb276d00a4368b6/tests/test_qubit_bench.py) or [test_curves.py](https://github.com/error-corp/qurveros/blob/1f53758e14e6be53cec63a159eb276d00a4368b6/tests/test_curves.py).